### PR TITLE
Fix shadowing variables

### PR DESCRIPTION
**Pull request by CharlesGueunet**
Fix some shadowing and add the corresonding gcc flag in CMakeLists. Also update the indent of the file.

### CODE REVIEW

**Comment by CharlesGueunet** [comment]
Is `src/F3DLoader.cxx` indented correctly ? I  have reindented it in a separate commit. Can drop if not necessary

**Comment by mwestphal** [comment]
> Is `src/F3DLoader.cxx` indented correctly ? I have reindented it in a separate commit. Can drop if not necessary

We dont have a clang format, but we should, your hanges are fine.

**Review by mwestphal — APPROVED** [review]

**Comment by mwestphal** [comment]
codecov report is incorrect, yet again. you can ignore it. ready to merge @CharlesGueunet ?

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -250,7 +250,7 @@ if(F3D_STRICT_BUILD)
     # Warning C4275 is essentially noise, disabling it to silent an issue with jsoncpp library
     target_compile_options(libf3d PUBLIC /W4 /WX /wd4275)
   else()
-    target_compile_options(libf3d PUBLIC -Wall -Wextra -Werror)
+    target_compile_options(libf3d PUBLIC -Wall -Wextra -Wshadow -Werror)
```
Comment (mwestphal) [inline src/CMakeLists.txt:253]: you should add it on all targets, especially f3d exec

**Review by mwestphal — CHANGES_REQUESTED** [review]
not ready

**Review by mwestphal — APPROVED** [review]

**Pull request merged**